forch/faucet_state_collector.py

```diff
@@ -1350,7 +1368,11 @@ def _get_egress_port(self, switch):
         return None
 
     def _is_egress_port(self, switch, port):
```

Comment:
The change fixes the error observed in prod where LAG state returns None. Seems that's the case when lag state is restored form varz.